### PR TITLE
ODF template refresh

### PR DESCRIPTION
**Pull request by gayathrimenath-ibm**

### CODE REVIEW

**Review by akgunjal — APPROVED** [review]
lgtm

File: config-templates/redhat/odf-local/changelog.json [hunk 1]
```diff
@@ -3,114 +3,105 @@
...
-                "golang": "1.18.9",
-                "ubi": "8.7-1049.1675784874",
-                "new-features": [],
-                "fixes": []
             }
         }
     },
     {
         "4.8": {
             "status": "supported",
```
Comment (mssachan) [inline config-templates/redhat/odf-local/changelog.json]: change status `supported` ---> `deprecated`

File: config-templates/redhat/odf-local/changelog.json [hunk 2]
```diff
@@ -3,114 +3,105 @@
...
-                "fixes": []
             }
         }
     },
     {
         "4.7": {
-            "status": "deprecated",
-            "latest-revision": {
+            "status": "unsupported",
+            "previous-revision": {
```
Comment (mssachan) [inline config-templates/redhat/odf-local/changelog.json]: No need to have any revision info once template is unsupported
Refer this: https://github.com/IBM/ibm-satellite-storage/blob/develop/config-templates/ibm/ibm-vpc-block-csi-driver/changelog.json#L27

File: config-templates/redhat/odf-remote/changelog.json [hunk 3]
```diff
@@ -3,114 +3,105 @@
...
-                "golang": "1.18.9",
-                "ubi": "8.7-1049.1675784874",
-                "new-features": [],
-                "fixes": []
             }
         }
     },
     {
         "4.8": {
             "status": "supported",
```
Comment (mssachan) [inline config-templates/redhat/odf-remote/changelog.json]: `supported` --> `deprecated`

File: config-templates/redhat/odf-remote/changelog.json [hunk 4]
```diff
@@ -3,114 +3,105 @@
...
-                "fixes": []
             }
         }
     },
     {
         "4.7": {
-            "status": "deprecated",
-            "latest-revision": {
+            "status": "unsupported",
+            "previous-revision": {
```
Comment (mssachan) [inline config-templates/redhat/odf-remote/changelog.json]: same here

**Review by mssachan — CHANGES_REQUESTED** [review]

**Review by mssachan — APPROVED** [review]
LGTM

**Pull request merged**